bin/brew

```diff
@@ -64,7 +64,7 @@ HOMEBREW_LIBRARY="${HOMEBREW_REPOSITORY}/Library"
 
 # Copy and export all HOMEBREW_* variables previously mentioned in
 # manpage or used elsewhere by Homebrew.
-for VAR in BROWSER DISPLAY EDITOR NO_COLOR PATH TMUX
+for VAR in BROWSER DISPLAY EDITOR NO_COLOR PATH TMUX DBUS_SESSION_BUS_ADDRESS
```

Comment:
Are you planning to make use of a `HOMEBREW_DBUS_SESSION_BUS_ADDRESS` (or maybe `EnvConfig.dbus_session_bus_address`) variable? I couldn't tell from your other PR.

Reply:
Yes, the current variable there will be replaced by `ENV["DBUS_SESSION_BUS_ADDRESS"] = ENV["HOMEBREW_DBUS_SESSION_BUS_ADDRESS"]`